website/docs/releases.md

```diff
@@ -21,6 +21,8 @@ In case you are wondering why all our releases start with `0.0`, read [this FAQ
 
 :::
 
+* Support for fiscal calendar options for `EXTRACT`, `date_part` and `date_trunc` was added. See [Fiscal Calendar Options](/docs/sql/scalar_func/datetime#fiscal-calendar-options)
```

Comment:
not only for options, but also for the time units/parts `fiscal_quarter`, `fiscal_month` etc.

Reply:
added the fields